### ISSUE TITLE
Flat package view in VSCode Project explorer

### ISSUE THREAD

**Issue by nilskp** [scalameta/metals-feature-requests#44]
**Is your feature request related to a problem? Please describe.**
The hierarchical view lists sub-folders before types, which makes it harder to quickly view types in a package.

**Describe the solution you'd like**
A flat package view makes it much faster to find the right package and containing types.

**Describe alternatives you've considered**
None.

**Additional context**
Here's how it looks in Eclipse, where you can quickly see all packages, without the need to keep digging:

![image](https://user-images.githubusercontent.com/74843/63870541-c71b5e80-c97f-11e9-9d03-40848292d947.png)

**Search terms**:
flat, flatten, package,

**Comment by mudsam** [comment]
If ok I’ll take a dig at implementing this

**Comment by wtho** [comment]
A generic flat view just landed in vscode/master

https://github.com/microsoft/vscode/issues/41627

![image](https://user-images.githubusercontent.com/22350/68310269-01a42480-00b0-11ea-9bc7-00dc43369fea.gif)

**Comment by tgodzik** [comment]
Look like we can close it then! :tada: 

@nilskp 
Are you happy with having it this way in VS Code?

**Comment by nilskp** [comment]
@tgodzik

yeah, I was referring to Metals' own Build view, not the VSCode file hierarchy. 

Even the solution from VSCode seems to be conditional on there being a single child. I think that's a different use case entirely.

I'm mostly looking for a flat list of packages, sorted alphabetically, like the Eclipse screenshot.
